network_wrangler/Scenario.py

```diff
@@ -2,72 +2,129 @@
 # -*- coding: utf-8 -*-
 
 from __future__ import annotations
-
-import os
-
+import os, sys
 from .ProjectCard import ProjectCard
+from collections import OrderedDict
+from .Logger import WranglerLogger
 
 class Scenario(object):
     '''
     Holds information about a scenario
     '''
 
-    def __init__(self, base_scenario: dict, project_cards: [ProjectCard] = None):
+    def __init__(self, base_scenario: dict, project_cards: [ProjectCard] = None, prerequisite_dict: dict = None, corequisite_dict: dict = None, conflict_dict: dict = None):
         '''
         Constructor
 
         args:
         base_scenario: the base scenario
-        project_card: this scenario's project card
+        project_cards: this scenario's project cards
         '''
 
         self.base_scenario = base_scenario
         self.project_cards = project_cards
 
-        self.prerequisites = {} # dictionary of
-        self.corequisites  = {}
-        self.conflicts     = {}
+        self.prerequisites = prerequisite_dict
```

Comment:
I think this is initiated as an empty dict and then when projects are added, these are added